app/build.gradle

```diff
@@ -35,6 +35,11 @@ android {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            buildConfigField "String", 'API_KEY', YoutubeApiKey
+        }
+
+        debug {
+            buildConfigField "String", 'API_KEY', YoutubeApiKey
```

Comment:
Why not a single entry?

Also, use environment variables

Reply:
@iamareebjamal Android Studio was throwing errors for single entry.